src/Controller/Api/GroupController.php

```diff
@@ -23,9 +23,11 @@
 use FOS\RestBundle\Request\ParamFetcherInterface;
 use FOS\RestBundle\View\View as FOSRestView;
 use FOS\UserBundle\Model\GroupInterface;
-use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Nelmio\ApiDocBundle\Annotation\Model;
+use Nelmio\ApiDocBundle\Annotation\Operation;
 use Sonata\DatagridBundle\Pager\PagerInterface;
 use Sonata\UserBundle\Model\GroupManagerInterface;
+use Swagger\Annotations as SWG;
```

Comment:
Why not using `Swagger` as an alias or is this best practice?

Reply:
Becouse `nelmio/api-doc-bundle` contains command `SwaggerDocblockConvertCommand` which convert ApiDoc annotations to Swagger-PHP annotations. `SWG` is usage there: 
https://github.com/nelmio/NelmioApiDocBundle/blob/master/UPGRADE-3.0.md